jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -565,6 +565,7 @@ YUI.add('juju-controller-api', function(Y) {
             controllerUUID: result['controller-uuid'],
             owner: tags.parse(tags.USER, result['owner-tag']),
             credential: credential,
+            credentialName: this._parseCredentialName(credential),
```

Comment:
What the what? I did actually add the docstrings. What happened to them?! I guess I'll add them again :).